src/pages/admin/create-notes.tsx

```diff
@@ -0,0 +1,418 @@
+
+  render() {
+      if (this.props.authState === "signedIn") {
+          return (
+              <div>
+                  <IndexApp></IndexApp>
+              </div>
+          );
+      } else {
+          return null;
+      }
+  }
+}
+
+class IndexApp extends React.Component<Props, State> {
+  constructor(props : Props) {
+    super(props);
+    this.state = { 
+      // input
+      editorState: EditorState.createEmpty(),
+      date: null,
+      pdf: '',
+      title: 'Unlisted',
+
+      // tags
+      selectedTags: [],
+      currentTag: '',
+
+      // display state
+      showPreview: false,
+      moreOptions: false,
+      showEditModal: false,
+      showAlert: '',
+      dateWarning: '',
+
+      // imports
+      notesList: [],
+      noteEdit: null,
+      noteEditObject: null,
+
+      // the power to delete things
+      delete: '',
+      understand: '',
+
+      // upload object
+      noteObject: {}
+    }
+
+    this.listNotes(null);
+    this.handleEdit = this.handleEdit.bind(this);
+    this.handleSave = this.handleSave.bind(this);
+    this.handleDeleteNote = this.handleDeleteNote.bind(this);
+  }
+
+  listNotes(nextToken: any) {
+
+    var listNotess:any = API.graphql({
+        query: queries.listNotess,
+        variables: { nextToken: nextToken, sortDirection: "DESC", limit: 200 },
+        authMode: GRAPHQL_AUTH_MODE.API_KEY
+    });
+
+    listNotess.then((json: any) => {
+        console.log({ "Success customQueries.listNotess: ": json });
+        this.setState({
+              notesList: this.state.notesList.concat(json.data.listNotess.items).sort(function(a: any, b: any) {
+              var nameA = a.id.toUpperCase();
+              var nameB = b.id.toUpperCase();
+              if (nameA < nameB) {
+                return 1;
+              }
+              if (nameA > nameB) {
+                return -1;
+              }
+              return 0;
+            })
+        })
+        if (json.data.listNotess.nextToken != null)
+            this.listNotes(json.data.listNotess.nextToken)
+
+    }).catch((e: any) => { console.log(e) })
+  }
+
+  handleDeleteNote() {
+    if (this.state.understand === "Delete forever") {
+      var deleteNotes:any = API.graphql({
+          query: mutations.deleteNotes,
+          variables: { input: {'id': this.state.delete} },
+          authMode: GRAPHQL_AUTH_MODE.AMAZON_COGNITO_USER_POOLS
+      });
+
+      deleteNotes.then((json: any) => {
+          console.log({ "Success mutations.deleteNotes: ": json });
+          this.setState({
+            delete: '',
+            understand: '',
+            showAlert: '⚠️ Deleted'
+          })
+
+      }).catch((e: any) => { console.log(e) })
+      return true;
+    } else {
+      this.setState({ showAlert: '⚠️ You must type the confirmation message' })
+    }
+  }
+
+  handleSave() {
+    if (this.state.editorState.getCurrentContent().hasText() === false || this.state.date === null) {
```

Comment:
Don't compare to boolean literals, `!this.state.editorState.getCurrentContent().hasText()` is better.